.gitignore

```diff
@@ -1,5 +1,4 @@
 /bin/*
-!/bin/.gitkeep
```

Comment:
If we decide not to keep the bin folder, you can change the first line to `/bin`